src/effects/effectslot.cpp

```diff
@@ -304,6 +312,20 @@ void EffectSlot::loadEffect(const EffectManifestPointer pManifest,
         ++iNumParameter;
     }
 
+    for (unsigned int i=0 ; i<m_knobParameterSlots.size() ; ++i) {
```

Comment:
Remove this new code. It will not be needed when hidden parameters are not loaded to slots.